src/terminal/adapter/ITermDispatch.hpp

```diff
@@ -110,6 +110,8 @@ class Microsoft::Console::VirtualTerminal::ITermDispatch
     virtual bool SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle) = 0; // DECSCUSR
     virtual bool SetCursorColor(const COLORREF color) = 0; // OSCSetCursorColor, OSCResetCursorColor
 
+    virtual bool SetClipboard(std::wstring_view content) = 0; // OscSetClipboard

```

Comment:
```suggestion
    virtual bool SetClipboard(std::wstring_view content) = 0; // OSCSetClipboard
```

for naming consistency (when we inevitably get a tool that parses these comments)

Reply:
(Which is in the works back on the first page of pull requests :P)

Reply:
Fixed.